utf/src/main/kotlin/com/github/softwareAnalysisTeam/unitTestFuzzer/fuzzers/JQFZestFuzzer.kt

```diff
@@ -0,0 +1,293 @@
+package com.github.softwareAnalysisTeam.unitTestFuzzer.fuzzers
+
+import com.github.javaparser.StaticJavaParser
+import com.github.javaparser.ast.*
+import com.github.javaparser.ast.body.*
+import com.github.javaparser.ast.expr.*
+import com.github.javaparser.ast.stmt.*
+import com.github.javaparser.ast.type.*
+import com.github.softwareAnalysisTeam.unitTestFuzzer.CommandExecutor
+import com.github.softwareAnalysisTeam.unitTestFuzzer.Fuzzer
+import com.github.softwareAnalysisTeam.unitTestFuzzer.logger
+import java.io.*
+import java.nio.file.Files
+import java.nio.file.Paths
+
+const val CLASS_FOR_ZEST_FUZZING_NAME = "ClassForFuzzing"
+const val CLASS_FOR_ZEST_SAVING_NAME = "ClassForSaving"
+
+class JQFZestFuzzer(private val outputDir: String, private var cp: String, private val JQFDir: String) : Fuzzer {
+    private val commandToCompile: String
+    private val commandToRepr: String
+    private val JQFgeneratedValuesDirName = "JQFGeneratedValuesForTests"
+
+    init {
+        val fileForPaths = File(outputDir + File.separator + "JQFPaths")
+        fileForPaths.createNewFile()
+        CommandExecutor.execute("$JQFDir/scripts/classpath.sh", outputDir, fileForPaths)
+        this.cp = "$cp:${fileForPaths.readText()}:${Paths.get(".")}"
+        fileForPaths.delete()
+
+        commandToCompile = "javac -cp ${this.cp}"
+        commandToRepr = "${Paths.get(JQFDir, "bin", "jqf-repro")} -c ${this.cp}"
+    }
+
+    override fun getValues(
+        testingClassName: String,
+        packageName: String?,
+        cu: CompilationUnit,
+        seeds: Map<MethodDeclaration, List<Expression>>,
+        budgetPerMethod: Double
+    ): Map<String, List<String>> {
+        if (seeds.isEmpty()) {
+            logger.debug("Map with seeds is empty.")
+            return mutableMapOf()
+        }
+
+        val commandToRun = "timeout ${budgetPerMethod}s ${Paths.get(JQFDir, "bin", "jqf-zest")} -c ${this.cp}"
```

Comment:
Reallly hard bug (it doesn't work like described in zest tutorial (It's ok in just terminal, but not in a program)

Reply:
Describe the problem in comments